subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/java/JavaInstantExecutionPerformanceTest.groovy

```diff
@@ -46,7 +46,7 @@ class JavaInstantExecutionPerformanceTest extends AbstractCrossVersionGradleInte
     def "assemble on #testProject #action instant execution state with #daemon daemon"() {
 
         given:
-        runner.targetVersions = ["6.5-20200512182414+0000"]
+        runner.targetVersions = ["6.6-20200531220033+0000"]
```

Comment:
is this faster or slower? if it's slower, do we know why?

Reply:
For a few tests it's faster and for other tests it's nearly failing. In general, there's more data being written to the configuration cache since the last update.